README.md

```diff
@@ -1,14 +1,23 @@
 # Prepare Your ASP.NET Core Application for 🚀 Liftoff: Hardening Your Applications for Production
 
 ## Abstract
-How launch-ready is your ASP.NET Core architecture?  Can you handle an indeterminate scale over time?  Does your application report what's it doing overtime?  Can anyone on your team actually read the logs and create actionable feedback from them?  
 
-Is deployment an "all hands on deck" scenario?  Does deployment happen continuously, quarterly, or somewhere in between?  Is being on-call stressful and anxiety-provoking?
+You're about to push your latest project out to the launch pad - nearly every feature is complete, it's quick, and you're excited for launch day. But are you ready to achieve orbit and stay there safely?
+ * Will it still be responsive when everyone uses it at the same time?
+ * Do you have enough instrumentation in place to know if it's on course or heading for a crash?
+ * Once you find a problem, can you efficiently ship a fix?
 
-If you shook your head reading any of these questions, it is a good chance your applications are not launch-ready!
+In this full-day workshop, you'll be taken on a journey to build a launch-ready, battle-hardened ASP.NET Core application. You'll learn practical techniques you can put in place to:
 
-In this full-day workshop, you'll be taken on a journey to build launch-ready, battle-hardened ASP.NET Core Architectures.  You'll learn about tracking application health with Health Checks, logging application telemetry and diagnostic information, and tracking configuration and secrets across multiple environments.  Additionally, you'll receive hands-on experience with deployment automation, and building maintenance strategies only SpaceX would dream about!
+ * Design for scale without breaking your schedule.
+ * Track and understand application health
+ * Provide telemetry to your dev & ops team, product team, and marketing team.
+ * Deploy quickly and safely when you have updates and fixes
+ * Scale to handle surges of inactivity.
 
+.NET Core offers an incredible set of tools and strategies to solve these problems, but you can't (and shouldn't) try to employ them all. We'll guide you through making good decisions on what's worth it for your situation and what you can afford to leave back in the workshop.
```

Comment:
Should we say `.NET Core` or just go `.NET`, while implying .NET `Core` and newer

Reply:
ASP.NET Core is correct, but they're dropped Core from versions 5 and newer.

Reply:
I equivocate on that - we could say .NET 5, I'd rather not just say ".NET".

Reply:
`.NET 5 and newer`?  .NET 6 will be out by CodeMash

Reply:
Man, this pace of change is crazy.  They need to get away from the short term/long term release thing because it causes a lot of market confusion.

I think I prefer just ".NET" to ".NET 5 (and newer!)" or something like that.  But, I'm willing to go with what you think is right I suspect you have a better feel from a marketing-to-developers standpoint.  I tend to think too formally since mostly I write documentation.

Reply:
LOL.  They're already iterating on .NET 7 in the repos.

I think `.NET` is a safer play here.

Reply:
Question - did you de-MD the file?  I'm noting my CRLF went away on each line.  VS Code is picky about MD and harasses me when I don't follow that aspect of MD so I gave up a while ago and put CRLF at the end of each line around 80 chars...

Reply:
I changed it to .NET from .NET Core.

Reply:
aye.  maybe.  probably.  I did the edits directly in github.